dao/src/main/java/org/thingsboard/server/dao/sql/query/DefaultEntityQueryRepository.java

```diff
@@ -886,4 +890,27 @@ public static EntityType resolveEntityType(EntityFilter entityFilter) {
                 throw new RuntimeException("Not implemented!");
         }
     }
+
+    //    "component_descriptor" - not entity Type
+    private static Map<EntityType, String[]> getEntityNameColumnsForSearchText(){
+        Map<EntityType, String[]> entityNameColumns = new ConcurrentHashMap<>();
```

Comment:
Use TbPair instead of the array with two values. it will make code more readable. 
```
Map<EntityType, TbPair>
```